keybase.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "domain": "lobste.rs",
+  "display_name": "Lobsters",
+
+  "description": "Computing-focused community centered around link aggregation and discussion",
+
+  "brand_color": "#AC130D",
+
+  "logo": {
+    "svg_black": "https://lobste.rs/small-black-logo.svg",
```

Comment:
![logo](https://user-images.githubusercontent.com/6262/56664334-77f2b380-6697-11e9-8784-c35adb96d76c.png)

This is all we have. Font is Baskerville Semi-Bold if you want to create an svg.